package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
It's odd, sometimes my Atom doesn't catch all the issues... that's why I added eslint. I can remove that bloat, no problem.